training/HelloDeepSpeed/train_bert.py

```diff
@@ -463,7 +465,7 @@ def create_experiment_dir(checkpoint_dir: pathlib.Path,
     try:
         gitlog = sh.git.log("-1", format="%H", _tty_out=False, _fg=False)
         with (exp_dir / "githash.log").open("w") as handle:
-            handle.write(gitlog.stdout.decode("utf-8"))
```

Comment:
Why remove `stdout.decode("utf-8")`?

Reply:
Thanks, this seems to becasue I'm using a newer version of `sh==2.0.4`, which has different behavior with the version in `requirements.txt`, I'll revert this change.